test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -619,7 +620,7 @@ public void RankingLightGBMTest()
             Done();
         }
 
-        [Fact]
+        [NotArm32Fact("Not supported on Arm32")]
```

Comment:
What isn't supported on Arm32? FastTree? Don't we build it ourselves?

Reply:
So this one is actually an interesting case that may be good to sync on sometime. But basically we are hitting a case (only  when we combine trees so fast tree itself is fine) where RyuJit is actually not creating the right code. An issue has been created since 2017 in the runtime repo about this, but its been pushed back because no one has really encountered this issue before. https://github.com/dotnet/runtime/issues/7970. But our code is trying to access float arrays in an unaligned manner (I can get you the exact code thats causing the problem if you are curious) and support for that doesn't exist for Arm32.

Reply:
@BruceForstall - any thoughts here? Is there something we can do to work around this issue? Or can the RyuJit bug get fixed?

@michaelgsharp - let's not say "Not supported on Arm32" here, but instead link to that dotnet/runtime issue.

Reply:
Sounds good. I have made the change to reflect that.

Reply:
If we have an example of real-world code where RyuJIT is doing the wrong thing w.r.t. alignment, then by all means we should fix it. Feel free to update the linked bug (especially if you can give a reduced, standalone test case), and add back the "untriaged" tag to make sure we'll see it. My reading of the linked bug is that we haven't seen such a case before.

Reply:
@michaelgsharp - do you think you could update that bug with your findings here? It doesn't need to be done before merging this PR. But we should do it sometime to ensure this scenario gets fixed, since we want to say we support ML.NET on arm now.

Reply:
Yep. I'll update my findings after we get this PR merged. I'll also see if I can get a reduced standalone test case, but even if I can't I'll get all the info in there.